src/common/style/vanilla/icons.scss

```diff
@@ -0,0 +1,189 @@
+@import 'vanilla-framework/scss/settings';
+
+@mixin icon-size($size) {
```

Comment:
For now we do, as these styles are no apart of any NPM package. They will be changed once the conversion of the dashboard theme to vanilla begins